src/styles/design-tokens/colors/index.scss

```diff
@@ -1,4 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&family=Roboto&display=swap');
```

Comment:
Adicionar as fontes utilizando `link` tem melhor performance.

Reply:
Fiz a modificação em um novo commit mas surgiu uma dúvida, eu devo referenciá-lo aqui neste texto?

https://github.com/Viniciusgalmeida/challenge-001/pull/2/commits/114ca4394ab3958c78c7733bb01a0df79bfcf21d

Reply:
O ideal seria fazer squash para juntar os commits utilizando `rebase interactive`.

[Aprenda aqui](https://www.git-tower.com/learn/git/faq/git-squash#:~:text=To%20%22squash%22%20in%20Git%20means,stand%2Dalone%20git%20squash%20command)